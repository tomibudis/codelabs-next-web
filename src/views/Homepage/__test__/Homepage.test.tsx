import React from "react";
import { render, screen } from "~utils/index";

import Homepage from "../Homepage";

describe("<Homepage />", () => {
  it("renders a homepage", () => {
    render(<Homepage />);

    const heading = screen.getByText("Homepage");

    expect(heading).toBeInTheDocument();
    expect(heading).toMatchSnapshot();
  });
});
