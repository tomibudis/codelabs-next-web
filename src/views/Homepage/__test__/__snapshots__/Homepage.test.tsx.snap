// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage /> renders a homepage 1`] = `
<div>
  Homepage
</div>
`;
